tools/accuracy_checker/accuracy_checker/preprocessor/README.md

```diff
@@ -25,7 +25,7 @@ Accuracy Checker supports following set of preprocessors:
     - `fit_to_window` - adaptive resize keeping aspect ratio for fit image into window with fixed size `[dst_height x dst_width]`,
          but trying to make the image as big as possible.
     - `frcnn_keep_aspect_ratio` - adaptive resize keeping aspect ratio for fit image into window with fixed size `[max_size x max_size]`,
-         but trying to make the minimal dimension of image to be equal to `min_size` or as close to `min_size` as possible, where    
+         but trying to make the minimal dimension of image to be equal to `min_size` or as close to `min_size` as possible, where 
```

Comment:
This change shouldn't be in this PR. I would've let it slide, but you didn't even remove _all_ of the trailing spaces. :-)

Reply:
Now I delete all :) And in other files too